samples/extensions/vertex_dynamic_state/README.md

```diff
@@ -0,0 +1,209 @@
+  std::vector<VkVertexInputAttributeDescription> vertex_input_attributes_1 = {
+    vkb::initializers::vertex_input_attribute_description(
+        0, 0, VK_FORMAT_R32G32B32_SFLOAT, 0),                  // Position
+    vkb::initializers::vertex_input_attribute_description(
+        0, 1, VK_FORMAT_R32G32B32_SFLOAT, sizeof(float) * 3),  // Normal
+
+    VkPipelineVertexInputStateCreateInfo vertex_input_state_1 =
+      vkb::initializers::pipeline_vertex_input_state_create_info();
+    vertex_input_state_1.vertexBindingDescriptionCount = static_cast<uint32_t>(vertex_input_bindings_1.size());
+    vertex_input_state_1.pVertexBindingDescriptions = vertex_input_bindings_1.data();
+
+    vertex_input_state_1.vertexAttributeDescriptionCount = static_cast<uint32_t>(vertex_input_attributes_1.size());
+    vertex_input_state_1.pVertexAttributeDescriptions = vertex_input_attributes_1.data();
+
+    graphics_create_info_1.pVertexInputState = &vertex_input_state_1;
+
+  vkCreateGraphicsPipelines(get_device().get_handle(), VK_NULL_HANDLE, 1,
+                            &graphics_create_info, VK_NULL_HANDLE, &pipeline1);
+
+  // Second pipeline vertex data
+  // Binding description
+  std::vector<VkVertexInputBindingDescription> vertex_input_bindings_2 = {
+      vkb::initializers::vertex_input_binding_description(
+          0, sizeof(Vertex), VK_VERTEX_INPUT_RATE_VERTEX),
+  };
+
+  // Attribute descriptions
+  std::vector<VkVertexInputAttributeDescription> vertex_input_attributes_2 = {
+    vkb::initializers::vertex_input_attribute_description(
+        0, 0, VK_FORMAT_R32G32B32_SFLOAT, 0),                  // Position
+    vkb::initializers::vertex_input_attribute_description(
+        0, 1, VK_FORMAT_R32G32B32_SFLOAT, sizeof(float) * 6),  // Normal
+
+    VkPipelineVertexInputStateCreateInfo vertex_input_state_2 =
+      vkb::initializers::pipeline_vertex_input_state_create_info();
+    vertex_input_state_1.vertexBindingDescriptionCount = static_cast<uint32_t>(vertex_input_bindings_2.size());
+    vertex_input_state_2.pVertexBindingDescriptions = vertex_input_bindings_2.data();
+
+    vertex_input_state_2.vertexAttributeDescriptionCount = static_cast<uint32_t>(vertex_input_attributes_2.size());
+    vertex_input_state_2.pVertexAttributeDescriptions = vertex_input_attributes_2.data();
+
+    graphics_create_info_2.pVertexInputState = &vertex_input_state_2;
+
+    vkCreateGraphicsPipelines(get_device().get_handle(), VK_NULL_HANDLE, 1,
+                              &graphics_create_info_2, VK_NULL_HANDLE, &pipeline2); 
+
+    draw_model(model1, pipeline1);
+    draw_model(model2, pipeline2);
+  };
+```
+
+However, with dynamic vertex input state multiple pipelines can be reduce in amount because of possibility to change vertex
+bindings and attribute descriptions on runtime by calling `vkCmdSetVertexInputEXT` before `draw_model`.
+Extension extends 2 structures: 
+"VkVertexInputBindingDescription" to "VkVertexInputBindingDescription2EXT" 
```

Comment:
I think those should be single quotes like above, so the the functions are properly highlighted by markdown.

Reply:
Of course I want to use other braces. Thanks for spotting that. I have implemented this change.